bbc/simorgh
# WSTEAM1-1027 Uploader File Upload Live Region and aria Labels

Resolves JIRA [WSTEAM1-1027](https://jira.dev.bbc.co.uk/browse/WSTEAM1-1027)

Overall changes
======
Adds a Live region component that the file uploader will send text to, to be read out by the screen reader. Adds `aria-required` and `aria-invalid` logic, for the File field.

Code changes
======

- Adds a LiveRegion component and LiveRegionContext for Live region re-usability
- Updates the File in `validateFunctions`, to cater for invalid and required logic
- Adds copy for extra text that will be read out by screen readers, these will need translations eventually

Testing
======
1. Visit http://localhost.bbc.com:7081/somali/send/u130092370?renderer_env=test
2. Make sure logic adheres to what aria labels are added
3. Check when adding and removing files that the live region text updates

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
